tomoscan/tomoscan.py

```diff
@@ -846,9 +854,23 @@ def compute_frame_time(self):
                 'Mono12Packed': 8.20,
                 'Mono16': 12.34
             }
+            readout_margin = 1.025
```

Comment:
Can we set the default to 1.01, which is what it was before it was previously increased for the Oryx?